src/assets/locales/de/main.json

```diff
@@ -1464,6 +1464,29 @@
         "success": "Usecase document upload has been done successfully."
       }
     },
+    "adminCertificate": {
+      "headline": "Credential Request Overview",
+      "search": "...search for certificate type",
+      "tabs": {
+        "all": "All",
+        "open": "Open",
+        "confirmed": "Confirmed",
+        "declined": "Declined"
+      },
+      "sortOptions": {
+        "name": "By Name",
+        "date": "By Date"
+      },
+      "table": {
+        "crendentialType": "Type",
+        "companyInfo": "Company Info",
+        "certificate": "Certificate",
+        "document": "Document"
+      },
+      "approvedMessage": "Credential Request Accepted successfully.",
+      "declinedMessage": "Credential Request Declined successfully.",
+      "errorMessage": "Something went wrong!"
+    },
```

Comment:
translate these lines to German to reduce the duplication. If that is planned to be done at a later time, maybe put `de: ` in front of every value